docs/DeveloperGuide.md

```diff
@@ -270,58 +270,208 @@ _{Explain here how the data archiving feature will be implemented}_
+
+  Use case resumes from step 2.
+
+
+**5. Use case: UC05 - Help List**
+
+**Preconditions: Nil**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Professor requests from JerylFypManager to list all possible commands.
+2.  JerylFypManager shows the list of possible commands and format to be entered in.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* Nil
+
+
+**6. Use case: UC06 - List FYPs**
+
+**Preconditions: There exist FYP(s) in JerylFypManager**
+
+**MSS**
+
+1.  Professor requests for JerylFypManager to show the list of FYPs.
+2.  JerylFypManager shows a list of FYPs to the professor.
+
+    Use case ends.
+
+
+**Extensions**
+
+* 2a. There are currently no FYPs that the professor is overseeing.
+    * 2a1. JerylFypManager tells the professor that he is currently not overseeing any FYPs.
 
   Use case ends.
 
-* 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+**7. Use case: UC07 - Exit JerylFypManager**
+
+**Preconditions: Nil**
+
+**MSS**
+
+1.  Professor requests to exit JerylFypManager.
+2.  JerylFypManager shows exit message.
+3.  JerylFypManager application is closed.
+
+    Use case ends.
+
+**Extensions**
+
+* Nil
 
-      Use case resumes at step 2.
 
 *{More to be added}*
 
+--------------------------------------------------------------------------------------------------------------------
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+2.  User with above average typing speed for regular English text (i.e. not code, not system admin commands) should be 
+able to accomplish most of the tasks faster using commands than using the mouse.
+3.  The system should respond in at most 1 second.
+4.  A user with no coding background should be able to use JerylFypManager.
+5.  Notification sent to users upon updates to FYPs.
+6.  Personal details of students should only be accessible by supervising professor.
+7.  System should be able to support the whole SoC cohort (around 1500 users) at the same time.
+8.  JerylFypManager is accessible 24/7.
 
 *{More to be added}*
 
+--------------------------------------------------------------------------------------------------------------------
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+* **FYP**: Final Year Project taken by year 4 undergraduate SoC students
```

Comment:
Do suggest more/edit/improve on glossary terms

Reply:
Apologies for have the first 2 points there. Should be removed since not related to our project

Reply:
No worries!